resources/lang/fr/forms.php

```diff
@@ -0,0 +1,102 @@
+    'end_day'                                   => 'Jour de fin',
+    'day'                                       => 'le jour',
+    'date_time'                                 => 'date et l heure',
+    'images'                                    => 'Images',
+    'chose_option'                              => 'Choisissez l option',
+    'alertant'                                  => 'Alerte',
+    'is_the_lost_person'                        => 'Est la personne perdue',
+    'not_is_the_lost_person'                    => 'N est-ce pas la personne perdue',
+    'is_the_contact_person'                     => 'Est la personne de contact',
+    'not_is_the_contact_person'                 => 'N est pas la personne de contact',
+    'age'                                       => 'Âge',
+    'phone'                                     => 'Téléphone',
+    'address'                                   => 'Adresse',
+    'incident'                                  => 'Incident',
+    'upa'                                       => 'Vu dernièrement à',
+    'village_last_place_seen'                   => 'village dernier endroit vu ',
+    'date_last_place_seen'                      => 'date du dernier endroit vu',
+    'incident_zone'                             => 'Zone d incident',
+    'possible_route'                            => 'Itinéraire possible',
+    'description'                               => 'La description',
+    'lost_people'                               => 'Les gens perdus',
+    'n_lost_people'                             => 'Nombre de personnes perdues',
+    'status_people'                             => 'Statut des personnes perdues',
+    'equipment_and_experience'                  => 'Équipement et expérience',
+    'knows_the_zone'                            => 'Connaît la zone',
+    'not_knows_the_zone'                        => 'Ne connaît pas la zone',
+    'bring_medication'                          => 'A des médicaments',
+    'not_bring_medication'                      => 'N a pas de médicament',
+    'bring_cold_clothes'                        => 'A des vêtements chauds',
+    'not_bring_cold_clothes'                    => 'N a pas de vêtements chauds',
+    'experience_with_activity'                  => 'A de l expérience avec l activité',
+    'not_experience_with_activity'              => 'N a pas d expérience avec l activité',
+    'bring_water'                               => 'A de l eau',
+    'not_bring_water'                           => 'N a pas d eau',
+    'bring_food'                                => 'A de la nourriture',
+    'not_bring_food'                            => 'N a pas de nourriture',
+    'bring_light'                               => 'A la lumière',
+    'not_bring_light'                           => 'N a pas de lumière',
+    'bring_waterproof_clothes'                  => 'A des vêtements imperméables',
+    'not_bring_waterproof_clothes'              => 'N a pas de vêtements imperméables',
```

Comment:
I think a "'" is missing between N and a

Reply:
Resoled the issues

Reply:
Resolved